scripts/criu-ns

```diff
@@ -83,44 +79,22 @@ def wrap_restore():
 			raise OSError(_errno, errno.errorcode[_errno])
 
 		# Spawn CRIU binary
-		criu_pid = os.fork()
-		if criu_pid == 0:
-			run_criu()
-			raise OSError(errno.ENOENT, "No such command")
-
-		while True:
-			try:
-				(pid, status) = os.wait()
-				if pid == criu_pid:
-					status = os.WEXITSTATUS(status)
-					break
-			except OSError:
-				status = -251
-				break
-
-		os.write(w_pipe, b"%d" % status)
-		os.close(w_pipe)
-
-		if status != 0:
-			sys.exit(status)
-
-		while True:
-			try:
-				os.wait()
-			except OSError:
-				break
```

Comment:
The init process has to reap any zombies and this loop did this work. Who will do it now?

Reply:
`criu` does that by default (unless `--restore-detached` is used)?
```bash
# criu/cr-restore.c:2448
	if (!opts.restore_detach && !opts.exec_cmd)
		wait(NULL);
```